test/functional/bulk_tests.js

```diff
@@ -1497,7 +1495,7 @@ describe('Bulk', function() {
   it(
     'should return an error instead of throwing when an empty bulk operation is submitted (with promise)',
     {
-      metadata: { requires: { promises: true, node: '>0.12.0' } },
+      metadata: { requires: { promises: true } },
```

Comment:
we can also remove all `promises` metadata, since we no longer support versions of node that don't provide a `Promise` implementation.